eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutableObjectPrimitiveHashMap.stg

```diff
@@ -328,6 +331,23 @@ final class ImmutableObject<name>HashMap\<K> extends AbstractImmutableObject<nam
         return this.delegate.keyValuesView();
     }
 
+    <if(!primitive.booleanPrimitive)>
```

Comment:
Is there a reason that some templates use primitive.booleanPrimitive and some use the string template map to avoid booleans?

Reply:
Just the convention which was followed in that particular template file.